src/views/CompletePage/style.css.ts

```diff
@@ -90,13 +199,26 @@ const pointBox = style({
   display: 'flex',
   justifyContent: 'center',
   alignItems: 'center',
-  width: 28,
-  height: 36,
   borderRadius: 6,
   transition: 'all 0.3s ease',
```

Comment:
transition이 all로 되어 있어서 width와 height도 0.3s에 걸쳐 변하고 있어요

https://github.com/user-attachments/assets/a96e8f3f-a632-4171-865b-6b7471de0498

```suggestion
  transition: 'color 0.3s ease, background-color 0.3s ease',
```
color와 background color만 적용되도록 수정하면 될 거 같아요!